indico/modules/events/tracks/templates/management.html

```diff
@@ -1,20 +1,35 @@
-{% extends 'layout/full_content_base.html' %}
+{% extends 'layout/base.html' %}
+{% from 'events/tracks/_track_list.html' import render_track_list %}
 
 {% block title %}
     {%- trans %}Programme{% endtrans -%}
 {% endblock %}
 
-{% block content %}
-    {% for track in tracks %}
-        <div>
-            {{ track.title }}
-        </div>
-    {% endfor %}
+{% block description %}
+    {%- trans -%}
+        Here you can find a list of tracks for this event.
+        You can add, edit, remove or sort these tracks.
+    {%- endtrans -%}
+{% endblock %}
 
+{% block content %}
     <a class="i-button highlight icon-plus"
         data-href="{{ url_for('.create', event) }}"
+        data-title="{% trans %}Add new track{% endtrans %}"
         data-ajax-dialog
         data-reload-after>
         {% trans %}Add track{% endtrans %}
     </a>
+        {% if not event.tracks %}
```

Comment:
Try adding a track to an event with no tracks. I think you need to use a container element around both this message and the UL that has the ID (you can then move both this part and the `<ul>` into the macro)